spec/visual/templates/layout.erb

```diff
@@ -38,18 +38,164 @@
     #page-footer {
       border-top: 1px dotted
     }
+
+    #theme-selector {
+      display: inline-block;
+      position: relative;
+      margin: 0 auto;
+      width: 170px;
+      font-family: Arial;
```

Comment:
Can this default to the system font? I'm not sure what the keyword for that is in CSS off the top of my head but it should be whatever will produce 'San Francisco' for Apple platforms.